.github/scripts/torchao_model_releases/README.md

```diff
@@ -20,8 +19,11 @@ Examples:
 
 Note: for initial release, please include `--populate_model_card_template` to populate model card template.
 
+### SMOOTHQUANT-W8A8
```

Comment:
can you add the command to generate smoothquant checkpoints as well? similar to AWQ-INT4